src/components/form-elements/radio/Radio.stories.mdx

```diff
@@ -0,0 +1,370 @@
+                    style={{
+                      backgroundColor: 'white',
+                    }}
+                  >
+                    <Headline bold color="text-gray-50" size="xsmall">
+                      State
+                    </Headline>
+                    <Headline
+                      extraBold
+                      transform="uppercase"
+                      type="span"
+                      color="text-gray-70"
+                      size="small"
+                    >
+                      {RadioType.type}
+                    </Headline>
+                  </td>
+                  <td>
+                    <Flex alignItems="center" direction="column">
+                      <Radio {...RadioType.props} errorMessage={null} />
+                      <Radio {...RadioType.props} checked errorMessage={null} />
+                    </Flex>
+                  </td>
+                  <td>
+                    <Flex alignItems="center" direction="column">
+                      <Radio {...RadioType.props}>Radio label</Radio>
+                      <Radio {...RadioType.props} checked>
+                        Radio label
+                      </Radio>
+                    </Flex>
+                  </td>
+                  <td>
+                    <Flex
+                      alignItems="center"
+                      direction="column"
+                      style={{
+                        width: '320px',
+                      }}
+                    >
+                      <Radio
+                        {...RadioType.props}
+                        description="More detailed description about this element. You can
+                          use here even some formatting and links."
+                      >
+                        Radio label
+                      </Radio>
+                      <Radio
+                        {...RadioType.props}
+                        description="More detailed description about this element. You can
+                          use here even some formatting and links."
+                        checked
+                      >
+                        Radio label
+                      </Radio>
+                    </Flex>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </StoryVariantTable>
+        </StyledTableWrapper>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### With external label
+
+<Canvas>
+  <Story name="With external label" args={{value: 'test'}}>
+    {args => (
+      <label id="myRadioID-label">
+        <Flex
+          style={{
+            backgroundColor: 'lightblue',
+            borderRadius: '8px',
+            padding: '20px 40px',
+          }}
+          alignItems="center"
+        >
+          This component is a label for radio
+          <Radio value={args.value} aria-labelledby="myRadioID-label" />
+        </Flex>
+      </label>
+    )}
+  </Story>
+</Canvas>
+
+### With custom theme
```

Comment:
I think we should avoid radio without label with our exaples. In this exampe we're showing color changes so first item is not needed.

The only case for radio without label is the last example of custom container.

![CleanShot 2022-06-07 at 17 42 58](https://user-images.githubusercontent.com/8572321/172423616-795d899e-c84d-49cd-99d5-5e232c1b178f.png)